torchsnapshot/storage_plugins/fs.py

```diff
@@ -17,7 +17,7 @@
 
 
 class FSStoragePlugin(StoragePlugin):
-    def __init__(self, root: str) -> None:
+    def __init__(self, root: str, storage_options: Dict[str, Any]) -> None:
```

Comment:
If we use `**storage_options`, `FSStoragePlugin` can remain `def __init__(self, root:str)`.